examples/test_pcbbot.py

```diff
@@ -220,7 +220,6 @@ def refinements(self) -> Refinements:
         (CompactKeystone5015, ['lcsc_part'], 'C5199798'),  # RH-5015, which is actually in stock
 
         # the camera recommended specs are excessively tight, so loosen them a bit
-        (Ov2640_Fpc24, ['device', 'dovdd', 'voltage_limits'], Range(1.71, 4.5)),
         (Ov2640_Fpc24, ['device', 'dvdd', 'voltage_limits'], Range(1.1, 1.36)),  # allow 1v2
```

Comment:
If this is deleted will it still work? Since this change also fixes the 1.2v limit

Reply:
yeah removing `  (Ov2640_Fpc24, ['device', 'dvdd', 'voltage_limits'], Range(1.1, 1.36)),  # allow 1v2` still works with no error. pushed with f0170fe1cef1b1cab850783d722505e51ea1171f